src/System.Net.Security/src/System/Net/NTAuthentication.cs

```diff
@@ -23,12 +23,12 @@ internal class NTAuthentication
         private string _clientSpecifiedSpn;
 
         private int _tokenSize;
-        private Interop.SspiCli.ContextFlags _requestedContextFlags;
-        private Interop.SspiCli.ContextFlags _contextFlags;
+        private ContextFlagsPal _requestedContextFlags;
+        private ContextFlagsPal _contextFlags;
 
         private bool _isCompleted;
         private string _protocolName;
-        private SecSizes _sizes;
+        private object _sizes;
```

Comment:
@shrutigarg, if ```NegoState.Encrypt``` is the only consumer, we may be able to remove this property and add the required code there.